www/src/data/tags-docs.js

```diff
@@ -0,0 +1,89 @@
+export const TAGS_AND_DOCS = new Map([
+  [`accessibility`, `/docs/making-your-site-accessible/`],
```

Comment:
We should document this somewhere, since it wasn't immediately clear to me what it was for. I was also worried these hard-coded links would be brittle – particularly the headings (which perhaps should be pulled into their own docs if they're that important, e.g. `#client-only-routes--user-authentication`?). 

To document, we could add a mention of blog tags & docs on the Gatsby Style Guide and perhaps in https://www.gatsbyjs.org/contributing/blog-and-website-contributions/ and https://www.gatsbyjs.org/contributing/docs-contributions/

Reply:
I agree that the headings could be brittle and moving them into their own guides seems like a good solution. Do you think that falls into the scope of a separate PR though?

I just made some updates to comment and document how to add tags, and how the whitelist works.

Reply:
New guides can definitely go in separate PRs! It would be good to file issues for them, however. Here are the ones I think could go in separate docs; if you've got any time to file issues for them that would be amazing!

- client-only-routes--user-authentication
- construct-new-higher-level-web-building-blocks

I think that's it. I left comments on the other ones!

Reply:
I'll go in and add issues, went down a rabbit hole finding missing pages from the sidebar when I started though woops